assignments/objects.js

```diff
@@ -19,6 +19,41 @@ const example = {
 
 // Write your intern objects here:
 
+const intern_0 ={
+  'id': 0,
+  'name': 'Mitzi',
+  'email:': 'mmelloy0@psu.edu',
+  'gender': 'F'
+}
+
+const intern_1 = {
+  id: 1,
+  name: 'Kennan',
+  email: 'kdiben1@tinypic.com' ,
+  gender: 'M'
+}
```

Comment:
```suggestion
speak: function() {
    return `Hello, my name is ${this.name}!`;
  }
}
```
You could also declare the method here in the object, rather than below before you call it. This is a little better for readability for the next developer who might work on this.